pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraItems.scala

```diff
@@ -85,29 +87,27 @@ object SierraItems extends Logging with SierraLocation with SierraQueryOps {
               case other => (other, loc.name)
             }
         }
-        .toSeq
         .distinct
 
     val fallbackLocation = otherLocations match {
       case Seq((Some(locationType), label)) => Some((locationType, label))
       case _                                => None
     }
 
-    sierraItemDataMap.values
+    itemDataEntries
       .foreach { itemData =>
         require(!itemData.deleted)
         require(!itemData.suppressed)
       }
 
-    val items = sierraItemDataMap.map {
-      case (itemId, itemData) =>
-        transformItemData(
-          bibId = bibId,
-          itemId = itemId,
-          itemData = itemData,
-          bibData = bibData,
-          fallbackLocation = fallbackLocation
-        )
+    val items = itemDataEntries.map { itemData =>
+      transformItemData(
+        bibId = bibId,
+        itemId = itemData.id,
+        itemData = itemData,
```

Comment:
Minor: let's simplify this function so it only takes `itemData`, and then we pick up the `.id` inside the function itself.